src/platforms/common/configuration/filtering.mdx

```diff
@@ -4,6 +4,7 @@ sidebar_order: 60
 description: "Learn more about how to configure your SDK to filter events reported to Sentry."
 notSupported:
   - perl
+  - kotlin-multiplatform
 ---
 
 Adding Sentry to your app gives you a great deal of very valuable information about errors and performance you wouldn't otherwise get. And lots of information is good -- as long as it's the right information, at a reasonable volume.
```

Comment:
```suggestion
When you add Sentry to your app, you get a lot of valuable information about errors and performance. And lots of information is good -- as long as it's the right information, at a reasonable volume.
```

Reply:
This bit of text hasn't been modified by my PR and the change also affects all other platforms. Maybe this change is better off in another PR?